tests/test_source.py

```diff
@@ -127,6 +128,20 @@ def test_get_package_hashes_server(self):
              'sha256': '795a1bdddc832289ab958dc9bd9a1c46b849011cbc81fd89d6fe144efc7aae69'}
         ]
 
+        # Testing get_package_hashes with included files
+        with open(os.path.join(self.data_dir, 'tensorflow-1.13.json')) as json_file:
+            result = json.load(json_file)
+            
+            aicoe_index = {
+                'name': 'aicoe',
+                'url': 'https://tensorflow.pypi.thoth-station.ninja/index/fedora29/jemalloc/simple',
+                'verify_ssl': True,
+                'warehouse': False
+            }
+
+            source = Source.from_dict(aicoe_index)
+            assert source.get_package_hashes('tensorflow', '1.13', True) == result
```

Comment:
Tests depend too much on external resources - on our index which might be down, inaccessible, can be moved or wheel file might change... Please get a small wheel Python package and include it in the repo (`tests/data` dir) and mock package retrieval.